ENIGMAsystem/SHELL/Bridges/SDL-OpenGL/graphics_bridge.cpp

```diff
@@ -57,6 +58,7 @@ void EnableDrawing(void*) {
 }
 
 void DisableDrawing(void*) {
+  cleanup_shaders(); // delete shaders before context
```

Comment:
Really, you could just add `graphicssystem_cleanup()` like the audio systems have rather than copying this to every bridge.
https://github.com/enigma-dev/enigma-dev/blob/f84a6a553ad1e8cc2796e5648582b8632f8fd04c/ENIGMAsystem/SHELL/Graphics_Systems/OpenGL-Common/std.cpp#L51

Reply:
I couldve if this was consistent between ogl n d3d but its not